Panic when executing qtsetup

HI,

I tried to recompile qt binding with latest changes but I got panic error when executing `qtsetup`.

```
AVAKRAIMA:Tools arknable$ qtsetup 
INFO[0000] running setup/prep                           
INFO[0000] running setup/check desktop                  
INFO[0000] VERSION: 0.0.4                               
INFO[0000] GOOS: darwin                                 
INFO[0000] GOARCH: amd64                                
INFO[0000] GOVERSION: go1.7.4                           
INFO[0000] GOROOT: /usr/local/Cellar/go/1.7.4_1/libexec 
INFO[0000] GOPATH: /Users/arknable/Documents/Cats/Repository/go 
INFO[0000] GOBIN: /Users/arknable/Documents/Cats/Repository/go/bin 
INFO[0000] HASH: 827c5c37dab564736dc0b7f0d606a9e287de7218 
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x2c4de0, 0xc42000c0e0)
    /usr/local/Cellar/go/1.7.4_1/libexec/src/runtime/panic.go:500 +0x1a1
github.com/therecipe/qt/internal/utils.QT_VERSION(0xc4200a20a0, 0xc42004bcf8)
    /Users/arknable/Documents/Cats/Repository/go/src/github.com/therecipe/qt/internal/utils/env.go:52 +0x28a
github.com/therecipe/qt/internal/cmd/setup.check(0x308ab2, 0x7)
    /Users/arknable/Documents/Cats/Repository/go/src/github.com/therecipe/qt/internal/cmd/setup/check.go:30 +0xb04
github.com/therecipe/qt/internal/cmd/setup.Setup(0x3078d9, 0x4, 0x308ab2, 0x7)
    /Users/arknable/Documents/Cats/Repository/go/src/github.com/therecipe/qt/internal/cmd/setup/setup.go:7 +0x136
main.main()
    /Users/arknable/Documents/Cats/Repository/go/src/github.com/therecipe/qt/cmd/qtsetup/qtsetup.go:37 +0x27b
```

Is it a possible bug ?

Thanks.

--- a/internal/utils/env.go
+++ b/internal/utils/env.go
@@ -49,7 +49,7 @@ func QT_VERSION() string {
 	componentsFilename := filepath.Join(filepath.Clean(dir), "components.xml")
 	fh, err := os.Open(componentsFilename)
 	if err != nil {
-		Log.Debugf(err, "couldn't open %s", componentsFilename)
+		Log.WithError(err).Debugf("couldn't open %s", componentsFilename)
 		return defaultVersion
 	}
 	defer fh.Close()

--- a/internal/utils/env.go
+++ b/internal/utils/env.go
@@ -59,13 +59,13 @@ func QT_VERSION() string {
 	}{}
 
 	if err = xml.NewDecoder(fh).Decode(cStruct); err != nil {
-		Log.Warnf("Couldn't decode XML: %s", err.Error())
+		Log.Warnf("Couldn't decode %s XML: %s", componentsFilename, err.Error())
 		return defaultVersion
 	}
 
 	words := strings.Split(cStruct.ApplicationName, " ")
 	if len(words) <= 1 {
-		Log.Warnf("Couldn't get valid application name '%s'", cStruct.ApplicationName)
+		Log.Warnf("Couldn't get valid application name '%s' in %s", cStruct.ApplicationName, componentsFilename)
 	}
 
 	QT_VERSION_CACHE = strings.TrimSpace(words[1])

--- a/internal/utils/env.go
+++ b/internal/utils/env.go
@@ -22,6 +22,8 @@ func QT_VERSION() string {
 		return version
 	}
 
+	const defaultVersion = "5.7.0"
+
 	//TODO: get version from tools instead ?
 	/*if UseHomeBrew() {
 		var (
@@ -39,23 +41,35 @@ func QT_VERSION() string {
 			return QT_VERSION_CACHE
 		}
 	} else {*/
-	if dir := os.Getenv("QT_DIR"); dir != "" {
-		dir = filepath.Clean(dir)
-
-		var (
-			cStruct = &struct {
-				ApplicationName string `xml:"ApplicationName"`
-			}{}
-			err = xml.Unmarshal([]byte(LoadOptional(filepath.Join(dir, "components.xml"))), cStruct)
-		)
-		if err == nil {
-			QT_VERSION_CACHE = strings.TrimSpace(strings.Split(cStruct.ApplicationName, " ")[1])
-			return QT_VERSION_CACHE
-		}
+	dir := os.Getenv("QT_DIR")
+	if dir == "" {
+		return defaultVersion
 	}
-	//}
 
-	return "5.7.0"
+	componentsFilename := filepath.Join(filepath.Clean(dir), "components.xml")
+	fh, err := os.Open(componentsFilename)
+	if err != nil {
+		Log.Debugf(err, "couldn't open %s", componentsFilename)
+		return defaultVersion
+	}
+	defer fh.Close()
+
+	cStruct := &struct {
+		ApplicationName string `xml:"ApplicationName"`
+	}{}
+
+	if err = xml.NewDecoder(fh).Decode(cStruct); err != nil {
+		Log.Warnf("Couldn't decode XML: %s", err.Error())
+		return defaultVersion
+	}
+
+	words := strings.Split(cStruct.ApplicationName, " ")
+	if len(words) <= 1 {
+		Log.Warnf("Couldn't get valid application name '%s'", cStruct.ApplicationName)
+	}
+
+	QT_VERSION_CACHE = strings.TrimSpace(words[1])
+	return QT_VERSION_CACHE
 }
 
 func QT_VERSION_MAJOR() string {

